server/banned.go

```diff
@@ -32,17 +32,23 @@ func (pi *packetInfo) checkBlameMessage() error {
 		return nil
 	}
 
+	reason := packet.GetMessage().GetBlame().GetReason()
```

Comment:
no logic change, but factored out `reason` to increase readability in 3 places below.